src/probabilities_estimators/histograms/rectangular_binning.jl

```diff
@@ -143,36 +141,11 @@ function RectangularBinEncoding(x::AbstractVector{<:Real}, b::RectangularBinning
     RectangularBinEncoding(b, mini, edgelength)
 end
 
-
-const RBE = RectangularBinEncoding
-const RB = RectangularBinning
-const NONDEDUCIBLE{T} = Union{
-    RB{T},
-    RBE{RB{T}}
-    } where T <: Union{Q, Vector{Q}} where Q <: AbstractFloat
-
-function total_outcomes(x, symbolization::NONDEDUCIBLE)
-    msg = "total_outcomes can't be deduced from $NONDEDUCIBLE"
-    throw(ArgumentError(msg))
-end
-
-# multiple-axis bins don't make sense for univariate input.
-function total_outcomes(::AbstractVector,symbolization::RBE{RB{Vector{Int}}})
-    msg = "total_outcomes is ambiguous for Vectors with RectangularBinning{Vector{Int}}"
-    throw(ArgumentError(msg))
-end
-total_outcomes(::AbstractVector,symbolization::RBE{RB{Int}}) =
-    symbolization.binning.ϵ
-total_outcomes(::AbstractDataset{D}, symbolization::RBE{RB{Int}}) where {D} =
-    symbolization.binning.ϵ^D
-total_outcomes(::AbstractDataset{D}, symbolization::RBE{RB{Vector{Int}}}) where {D} =
-    prod(symbolization.binning.ϵ)
-
 ##################################################################
 # Encoding bins using a fixed (user-specified) grid
 ##################################################################
 function RectangularBinEncoding(::AbstractVector{<:Real},
-        b::FixedRectangularBinning{E}; n_eps = 2) where E
+        b::FixedRectangularBinning{E}; n_eps = 1) where E
```

Comment:
Why did you change the tolerance from `n_eps = 2` to `n_eps = 1` here? The tests fail  before due to roundoff errors with lower tolerances.

Reply:
I thought this was a typo. The docstring says that we only do 1 call to `next_float`. I'll revert this then, but also update docstirng accordingly.